src/main/java/com/simibubi/create/content/kinetics/chainConveyor/ChainConveyorRenderer.java

```diff
@@ -83,6 +94,9 @@ private void renderBox(ChainConveyorBlockEntity be, PoseStack ms, MultiBufferSou
 
 		Vec3 position = physicsData.prevPos.lerp(physicsData.pos, partialTicks);
 		Vec3 targetPosition = physicsData.prevTargetPos.lerp(physicsData.targetPos, partialTicks);
+		if (frustum != null && !frustum.testSphere((float) (targetPosition.x - camPos.x), (float) (targetPosition.y - camPos.y), (float) (targetPosition.z - camPos.z), 1))
```

Comment:
How do we know the radius of the sphere is always 1? What if we add a very large package model?